tests/Feature/ExampleTest.php

```diff
@@ -4,18 +4,262 @@
 
 use Tests\TestCase;
 use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\DB;
 
 class ExampleTest extends TestCase
 {
-    /**
-     * A basic test example.
-     *
-     * @return void
-     */
-    public function testBasicTest()
+    static $member_created_id;
```

Comment:
Why are these properties `static`?

Reply:
i trying to declare a properties of class but it didnt save data through function. So when i declare static it's work

Reply:
OK that is because for every test case the class is instantiated. If you want to pass information from one test to another you can use the `@depends` annotation of phpunit:
https://phpunit.readthedocs.io/en/latest/annotations.html#depends

But if you use a database seeder and a separate database for testing you don't need it because the data will always be the same.

Reply:
thank you for instruction,